api/src/main/proto/stellarstation/api/v1/monitoring/monitoring.proto

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright 2018 Infostellar, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+import "stellarstation/api/v1/antenna/antenna.proto";
+import "stellarstation/api/v1/common/common.proto";
+import "stellarstation/api/v1/radio/radio.proto";
+
+package stellarstation.api.v1.monitoring;
+
+option go_package = "github.com/infostellarinc/go-stellarstation/api/v1/monitoring";
+
+option java_multiple_files = true;
+option java_outer_classname = "MonitoringProto";
+option java_package = "com.stellarstation.api.v1.monitoring";
+
+// Messages for reporting information about a ground station for monitoring and debugging.
+
+// The current configuration of a transmitter. A transmitter is taking a bitstream and applies
+// signal processing to create the final waveform sent to a satellite.
+message TransmitterConfiguration {
```

Comment:
A lot of these are CORTEX-specific (Same with `ReceiverConfiguration`). For a generic representation, I suggest we just use the existing `RadioDeviceConfiguration`. Then modify `RadioDeviceConfiguration` to have `line_coding`, `convolutional_coding_params` (can be disabled), `reed_solomon_params` (can be disabled), `scrambling_params` (can be disabled), `sync_word`.

Reply:
From what I understood, the the device monitoring API doesn't return many of these parameters, and mostly just booleans like I have here. Is there a different way for us to verify this information at the beginning of a pass? If so, I definitely would rather just use `RadioDeviceConfiguration`